tensorflow/compiler/mlir/hlo/include/mlir-hlo/Dialect/mhlo/IR/hlo_ops.td

```diff
@@ -2162,4 +2162,124 @@ def HLO_ReducePrecisionOp :
   let results = (outs HLO_FpTensor:$output);
 }
 
+def HLO_H2DOp: HLO_Op<"h2d", [NoSideEffect, SameOperandsAndResultType]> {
+  let summary = "H2D operator";
```

Comment:
Please expand, something like "Host-to-device copy operator" here.

Reply:
Mehdi, we take your advice.  Will add D2H, H2D and DebugPrint Ops to a new dialect. So remove H2D/D2H here.